src/indexed_crate.rs

```diff
@@ -2,21 +2,39 @@ use std::collections::HashMap;
 
 use rustdoc_types::{Crate, Id, Item, Visibility};
 
+/// Besides the parsed rustdoc, it additionally contains
+/// some manually inlined `rustdoc_types::Trait`s of
+/// most common built-in traits.
+/// This is a temporary step, until we'll adopt rustdocs of foreign crates.
```

Comment:
```suggestion
/// The rustdoc for a crate, together with associated indexed data to speed up common operations.
///
/// Besides the parsed rustdoc, it also contains some manually-inlined `rustdoc_types::Trait`s
/// of the most common built-in traits.
/// This is a temporary step, until we're able to combine rustdocs of multiple crates.
```
Small wording improvements + a headline that explains the type itself, since I felt it was a bit awkward to have a doc comment explaining a detail of the type without saying anything about the type as a whole.